Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:12
+
+ENV USER=root
+WORKDIR "/"
+
+COPY package.json ./
```

Comment:
You should be able to do all these copies in the same stage `COPY package.json yarn.lock truffle-config.js networks.json ./`

Reply:
I actually would go further and remove these `COPY`s, and instead moving the `COPY . .` statement here.

Reply:
@cag If you `COPY` everything nothing is cached, and that way when a single file is changed all the dependencies are fetched again. This way dependencies are just fetched when any of those files `package.json yarn.lock truffle-config.js networks.json` change